package.json

```diff
@@ -27,6 +27,8 @@
     "test:format": "prettier --check . --ignore-path .gitignore"
   },
   "dependencies": {
+    "@types/react-transition-group": "^4.4.5",
```

Comment:
Move this to `devDependencies` :pray:

Reply:
Done @nelsonmestevao , also just committed my last work on the Leaderboard feature, it's still a draft. Have a look and tell me what you think!
Also, I know I used that function `changeVariables()` with a time out for the transitions, but it was necessary for it to work! I tried having them both run at the same time using just one variable called `view` (where one transition would be activated by the `false` state and the other by the `true` state of this variable) but it doesn't work. I also tried to add a `transition-delay` to the CSS but had no luck with that either, but anyhow if you figure a way to avoid it hmu.

P.s. I know the `makeLeaderboard()` function is VERY inefficient but I'll improve it later kk.